packages/cli/src/commands/installedschema/delete.ts

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash'
+
+import { flags } from '@oclif/command'
+
+import { InstalledSchemaApp } from '@smartthings/core-sdk'
+
+import {selectAndActOn, APICommand, withLocations} from '@smartthings/cli-lib'
+
+
+export default class InstalledSchemaAppDeleteCommand extends APICommand {
+	static description = 'delete the installed schema connector instance'
+
+	static flags = {
+		...APICommand.flags,
+		'location-id': flags.string({
+			char: 'l',
+			description: 'filter results by location',
+			multiple: true,
+		}),
+		verbose: flags.boolean({
+			description: 'include location name in output',
+			char: 'v',
+		}),
+	}
+	static args = [{
+		name: 'id',
+		description: 'installed schema conntector UUID',
+	}]
+
+	primaryKeyName = 'isaId'
+	sortKeyName = 'appName'
+	protected listTableFieldDefinitions = ['appName', 'partnerName', 'partnerSTConnection', 'isaId']
```

Comment:
This really shouldn't be marked `protected`.  It is working but I'm not 100% sure how...I'm guessing it's because the `protected` is removed in the conversion to JavaScript. (The compiler is okay with it being protected because it sees the `sortKeyName` and `primaryKeyName` fields which it _thinks_ will be used. Then, at runtime the code sees this (because protected is gone) and uses it. That's my theory anyway.

Reply:
Removed protected